src/routes/message.ts

```diff
@@ -0,0 +1,45 @@
+import express from 'express';
+import MessageService from '../services/MessageService';
+import RequestMessage from '@/interfaces/RequestMessage';
+
+const router: express.Router = express.Router();
+const messageService: MessageService = new MessageService();
+
+router.post('/', (req, res) => {
+  const message: RequestMessage = {
+    platform: req.body.platform,
+    template: req.body.template,
+    payload: req.body.payload,
+    receivers: req.body.receivers,
+  };
+
+  if (req.body.sender) {
+    message.sender = req.body.sender;
+  }
+
+  if (typeof req.body.trackLinks === 'boolean') {
+    message.trackLinks = req.body.trackLinks;
+  }
+
+  messageService.send(message);
+
+  res.send('Message queued.');
+});
+
+router.post('/:messageId/seen', async (req, res) => {
+  if (!req.params.messageId) {
```

Comment:
Extract method for parameter checking to utils.

Reply:
@hefowe: Do you have any ideas how to do this without losing readability?

Reply:
We could use something like this: https://express-validator.github.io/docs/

Reply:
I created an Issue #34 for that and we should do that later.